test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -839,17 +842,22 @@ func (cont *NginxIngressController) Init() {
 	// --publish-service flag (see <IngressManifestPath>/nginx/rc.yaml) to make it work in private
 	// clusters, i.e. clusters where nodes don't have public IPs.
 	framework.Logf("Creating load balancer service for nginx ingress controller")
-	serviceJig := e2eservice.NewTestJig(cont.Client, "nginx-ingress-lb")
-	serviceJig.CreateTCPServiceOrFail(cont.Ns, func(svc *v1.Service) {
+	cont.svcJig = e2eservice.NewTestJig(cont.Client, "nginx-ingress-lb")
+	cont.svcJig.CreateTCPServiceOrFail(cont.Ns, func(svc *v1.Service) {
 		svc.Spec.Type = v1.ServiceTypeLoadBalancer
 		svc.Spec.Selector = map[string]string{"k8s-app": "nginx-ingress-lb"}
 		svc.Spec.Ports = []v1.ServicePort{
 			{Name: "http", Port: 80},
 			{Name: "https", Port: 443},
 			{Name: "stats", Port: 18080}}
 	})
-	svc := serviceJig.WaitForLoadBalancerOrFail(cont.Ns, "nginx-ingress-lb", e2eservice.LoadBalancerCreateTimeoutDefault)
-	serviceJig.SanityCheckService(svc, v1.ServiceTypeLoadBalancer)
+	cont.lbTimeout = e2eservice.LoadBalancerCreateTimeoutDefault
```

Comment:
There is already a method for this: GetServiceLoadBalancerCreationTimeout

Please use it (and then you can remove lbTimeout from the struct - it won't be needed).